<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNetsoul</class>
 <widget class="QMainWindow" name="QNetsoul">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QNetSoul</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Images.qrc">
    <normaloff>:/images/qnetsoul.png</normaloff>:/images/qnetsoul.png</iconset>
  </property>
  <property name="whatsThis">
   <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;About QNetSoul&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Version: beta 0.4&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;The first Netsoul's client built with the great framework Qt.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Feel free to send your opinion/comments on this software.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Thanks to:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; Pierre-Emmanuel Duc and Christophe Sundas for feedbacks.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;a href=&quot;http://www.famfamfam.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;FamFamFam&lt;/span&gt;&lt;/a&gt; for icons.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;a href=&quot;http://www.jonasraskdesign.com/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;Jonas Rask&lt;/span&gt;&lt;/a&gt; for icons.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Written by Richard Dally.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QComboBox" name="statusComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Online</string>
         </property>
         <property name="icon">
          <iconset resource="../Images.qrc">
           <normaloff>:/images/online.png</normaloff>:/images/online.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Locked</string>
         </property>
         <property name="icon">
          <iconset resource="../Images.qrc">
           <normaloff>:/images/lock.png</normaloff>:/images/lock.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Away</string>
         </property>
         <property name="icon">
          <iconset resource="../Images.qrc">
           <normaloff>:/images/away.png</normaloff>:/images/away.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Server</string>
         </property>
         <property name="icon">
          <iconset resource="../Images.qrc">
           <normaloff>:/images/server.png</normaloff>:/images/server.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="ContactsTree" name="tree">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_QNetSoul"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuQNetsoul">
    <property name="title">
     <string>&amp;QNetsoul</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>&amp;Contacts</string>
    </property>
    <addaction name="actionAddG"/>
    <addaction name="actionAddC"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionLoadContacts"/>
    <addaction name="actionSaveContacts"/>
   </widget>
   <widget class="QMenu" name="menuFeaturettes">
    <property name="title">
     <string>Fun</string>
    </property>
    <addaction name="actionVDM"/>
    <addaction name="actionCNF"/>
    <addaction name="actionPastebin"/>
   </widget>
   <addaction name="menuQNetsoul"/>
   <addaction name="menuAccount"/>
   <addaction name="menuFeaturettes"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPr_f_rences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAddC">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Add &amp;Contact</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionRemove_selected_contact">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove selected contact</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout_QNetSoul">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/information.png</normaloff>:/images/information.png</iconset>
   </property>
   <property name="text">
    <string>About QNetSoul</string>
   </property>
  </action>
  <action name="actionAbout_2">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/aboutQt.png</normaloff>:/images/aboutQt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/options.png</normaloff>:/images/options.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSaveContacts">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save contacts as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionToggle_sort_contacts">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/sort.png</normaloff>:/images/sort.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Toggle sort contacts</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionLoadContacts">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/load.png</normaloff>:/images/load.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load contacts...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh contacts</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionVDM">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/vdm.png</normaloff>:/images/vdm.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Vie de merde</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionPastebin">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/pastebin.ico</normaloff>:/images/pastebin.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Pastebin</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionCNF">
   <property name="icon">
    <iconset resource="../Images.qrc">
     <normaloff>:/images/chucknorris.png</normaloff>:/images/chucknorris.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Chuck Norris facts</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionAddG">
   <property name="text">
    <string>Add &amp;Group</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactsTree</class>
   <extends>QTreeWidget</extends>
   <header>ContactsTree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Images.qrc"/>
 </resources>
 <connections/>
</ui>
